src/test/transaction_tests.cpp

```diff
@@ -831,6 +831,7 @@ BOOST_AUTO_TEST_CASE(test_IsStandard)
     reason.clear();
     BOOST_CHECK(!IsStandardTx(CTransaction(t), reason));
     BOOST_CHECK_EQUAL(reason, "bare-multisig");
+    fIsBareMultisigStd = true;
```

Comment:
```suggestion
    fIsBareMultisigStd = DEFAULT_PERMIT_BAREMULTISIG;
```

Reply:
Alternatively, store the value at 824 and reload the stored value at 834.

Reply:
Agree that using the constant is clearer.